packages/firebase_admob/android/src/main/java/io/flutter/plugins/firebaseadmob/FirebaseAdMobPlugin.java

```diff
@@ -24,6 +27,12 @@
 
   RewardedVideoAdWrapper rewardedWrapper;
 
+  static Function<UnifiedNativeAd, View> nativeAdGenerator;
```

Comment:
Should this always build a UnifiedNativeAdView? (seems like it's a requirement according to: https://developers.google.com/admob/android/native/advanced )

Reply:
I don't believe it is required to return a UnifiedNativeAdView. I saw this as well and it only states `This class is a ViewGroup that publishers should use as the root for the UnifiedNativeAd`.

If it was required i would assume it would say `must`? There may be a situation where it isn't required, but I haven't looked further into it.

Reply:
Nvm. I missed a key point on the website that states it's required.